ts/project.ts

```diff
@@ -163,58 +179,80 @@ const proxyEnvironmentHelpers = (): ProjectHelpers => ({
     return this.defaultEnvironment.build(s, ...args);
   },
 
-  addExecutable<T extends Project, Name extends string>(this: T, name: Name) {
+  addExecutable<T extends Project, Name extends string>(
+    this: T,
+    name: Name,
+    script?: string,
+  ) {
     if (this.defaultEnvironment == null) {
       throw new Error(
         `'.addExecutable' can only be called on projects with a default environment`,
       );
     }
-    const templateLiteralFn = (
-      s: TemplateStringsArray,
-      ...args: Array<string>
-    ) => {
-      const newExecutable = { [name]: this.shell(s, ...args) } as {
-        [n in Name]: Executable;
-      };
+    if (script != null) {
+      const newExecutable = { [name]: this.shell(script) };
       return {
         ...this,
         ...newExecutable,
       };
-    };
-    markAsMayNotExport(templateLiteralFn, (exportName: string) =>
-      [
-        `${exportName} exports the return type of "addExecutable", but this is not the proper usage of addExecutable.`,
-        'Did you forget the template literal? Example usage: project.addExecutable("executable-name")`shell script to run`',
-      ].join(" "),
-    );
-    return templateLiteralFn;
+    } else {
```

Comment:
Nit, but maybe we remove this else? And the one in `addCheck` below?

Reply:
Done in fd157e61.